src/misc/lv_event.c

```diff
@@ -41,6 +44,10 @@
     #define LV_TRACE_EVENT(...)
 #endif
 
+#define EVENT_MARK_DELETING(list, dsc) do { list->has_marked_deleting = true; (dsc)->filter |= LV_EVENT_MARKED_DELETING; } while(0)
+#define EVENT_IS_MARKED_DELETING(dsc) (((dsc)->filter & LV_EVENT_MARKED_DELETING) != 0)
+#define EVENT_ARRAY(list) (&(list)->array)
```

Comment:
Emmm, Why not define an inline function for lv_event_array_size, lv_event_array_at.  Then most of the code uses EVENT_ARRAY could be simplified.

Reply:
in fact, code outer of the event module needn't do operations on event list.

Reply:
I think internal `static inline` would make it. They are much easier to debug.